### DISCUSSION TITLE
Welcome to Bliss Discussions!

### DISCUSSION THREAD

**Discussion by MrScautHD** [MrScautHD/Bliss#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zapturk** [comment]
Hello, I'm Zapturk a C# devloper. I currently make games with godot but I want to move to a Framework. I am looking forward to meeting other devs and helping to contribute to Bliss!

**Reply by MrScautHD** [reply]
Welcome Zapturk!
It’s awesome to hear you’re diving into frameworks.

**Reply by MrScautHD** [reply]
Note: Bliss is currently in its alpha stage, so it's not fully featured and may have a few bugs. However, we're making great progress toward a stable release!
